packages/cli/test/integration/shared/testDb.ts

```diff
@@ -514,11 +515,13 @@ export async function getWorkflowSharing(workflow: WorkflowEntity) {
 // ----------------------------------
 
 export async function createVariable(key: string, value: string) {
-	return Db.collections.Variables.save({
+	const result = await Db.collections.Variables.save({
```

Comment:
I'd expect `VariablesService` to offer `createVariable()` to take care of creating the var and updating the cache.